files/magneticmediajs/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Arnaud Leyder"
+github = "https://github.com/arnaudleyder"
```

Comment:
This is the project link not the author's link. So the correct is https://github.com/arnaudleyder/magneticmediajs